src/content/1.7/development/orders-lifecycle/_index.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Orders lifecycle
+weight: 42
+---
+
+# Order creation
+
+Orders can be created from a cart either in Front Office (FO) by the customer or in Back Office (
```

Comment:
link to a order page reference when merged https://github.com/PrestaShop/docs/pull/981

Reply:
https://github.com/PrestaShop/docs/pull/981 is merged 🎉